src/style.css

```diff
@@ -1 +1,40 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
```

Comment:
Don't use tag names for styling (except `html` and `body`). Style all elements using `.classes` and if needed with `:pseudo-classes`, `pseudo-elements` and `[attributes]`. For example, instead of styling `.stars`, if it's a `ul`, don't style it directly with `ul`.